src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -551,10 +552,27 @@ internal static List<CompletionResult> CompleteCommandParameter(CompletionContex
                 // Parent must be a command
                 commandAst = (CommandAst)dashAst.Parent;
                 partialName = string.Empty;
+
+                // If the user tries to tab complete a new parameter in front of a positional argument like: ls -<Tab> C:\
```

Comment:
Let's use `dir` instead of `ls` because on Unix platforms, `ls` is not an alias to `dir`, but the real `/bin/ls` executable.
```suggestion
                // If the user tries to tab complete a new parameter in front of a positional argument like: dir -<Tab> C:\
```